script/perturb_data.py

```diff
@@ -0,0 +1,81 @@
+import os
+import pandas as pd
+import numpy as np
+import argparse
+
+def perturb_graph(data_path, graph, perturbation_mode, k=0, seed=123):
+    np.random.seed(seed)
+
+    if graph == 'train1':
+        df = pd.read_csv(os.path.join(data_path, 'train1.txt'), sep='\t', header=None)
+    elif graph == 'train2':
+        df = pd.read_csv(os.path.join(data_path, 'train2.txt'), sep='\t', header=None)
```

Comment:
add if train0, that is composed of train1 and train2:


```suggestion
elif graph == "train0":
    df1 = pd.read_csv(os.path.join(data_path, 'train1.txt'), sep='\t', header=None)
    df2 = pd.read_csv(os.path.join(data_path, 'train1.txt'), sep='\t', header=None)
    df = pd.concat([df1, df2])
```

Reply:
I thought we decided to abandon the case of G0, but ok, let us have it.

Reply:
You're right, based on what we discussed we don't necessarily want this. Just keeping as a possibility